ReverseTicTacToe/UI/ConsoleUIManager.cs

```diff
@@ -0,0 +1,265 @@
+﻿using System;
+using System.Text;
+using static System.Console;
+using ReverseTicTacToe.Logic;
+using Ex02.ConsoleUtils;    
+
+namespace ReverseTicTacToe.UI
+{
+    class ConsoleUIManager
+    {
+        private const int k_MinBoardSize = 3;
+        private const int k_MaxBoardSize = 9;
+        private const int k_MinGameType = 1;
```

Comment:
I think these two gameType constants are redundant (i suggested a way not to use them in another comment). They should be managed exclusively by the Logic